docker/k8s/README.md

```diff
@@ -45,27 +46,31 @@ in SGX-SIM and SGX-HW mode.
     ```
 
     > The above command would run the 
-    > [propose_requests.sh](./scripts/propose_requests.sh) as a kubernetes job
+    > [propose_requests.sh](../../scripts/propose_requests.sh) as a kubernetes job
 
 3. Check the request handling logs (the request should be done in several 
    minutes after the proposal)
     ```bash
-    # Let's say 'propose-request-abcde' is the job created above
+    # Get workload request pod name using command
```

Comment:
Change "Get workload" to "Get the workload"

Reply:
Done